app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -810,7 +892,7 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
                         isValid = false
                     }
                 } catch (e: Exception) {
-                    Log.e(LOG_TAG, "$FILETAG Error: ${e.message}", e)
+                    Log.e(LOG_TAG_VPN, "$FILETAG Error: ${e.message}", e)
```

Comment:
don't really have to eat all exceptions here. Catch a specific exception; and if unclear, catch nothing.

Reply:
Modified to specific exception.